app/models/application_record.rb

```diff
@@ -8,14 +7,18 @@ module VERSION
         MAJOR = ActiveRecord::VERSION::MAJOR
       end
     end
-  end unless Rails.const_defined?("VERSION")
+  end if !defined?(::Rails) || !::Rails.const_defined?("VERSION")
   require 'acts_as_tenant'
 
   self.abstract_class = true
 
   def as_json(options = {})
     options[:except] ||= []
-    options[:except] << :tag_list
     super
   end
```

Comment:
We probably don't need to override this method at all now? cc @bdunne ?

Reply:
Yeah, that was because of `acts_as_taggable`